src/domain/dataset-update-flow/src/entities/flow/flow_trigger.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright Kamu Data, Inc. and contributors. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+use opendatafabric::{AccountID, AccountName, DatasetID};
+
+use crate::*;
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum FlowTrigger {
+    Manual(FlowTriggerManual),
+    AutoPolling(FlowTriggerAutoPolling),
+    Push(FlowTriggerPush),
+    InputDatasetFlow(FlowTriggerInputDatasetFlow),
+}
+
+/////////////////////////////////////////////////////////////////////////////////////////
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct FlowTriggerManual {
+    pub initiator_account_id: AccountID,
+    pub initiator_account_name: AccountName,
```

Comment:
We have to keep it for now, until we don't have impplemented `AccountID`.
Otherwise, UI will not be able to properly print the initiating accounts, task owners, ...

Reply:
Let's create a ticket for `AccountID`s if we don't have one yet and mention removing this field innnnnn it.

Reply:
We have an [epic ](https://github.com/kamu-data/kamu-cli/issues/244)